x/programs/rust/examples/counter/src/lib.rs

```diff
@@ -28,19 +28,6 @@ pub fn inc(context: Context<StateKeys>, to: Address, amount: Count) -> bool {
     true
 }
 
-/// Increments the count at the address by the amount for an external program.
-#[public]
-pub fn inc_external(
-    _: Context,
-    target: Program,
-    max_units: Gas,
-    of: Address,
-    amount: Count,
-) -> bool {
-    let args = borsh::to_vec(&(of, amount)).unwrap();
-    target.call_function("inc", &args, max_units, 0).unwrap()
```

Comment:
this was our old way of calling external programs, and we have an example of how to do it in `counter-external`